node/src/rpc.rs

```diff
@@ -47,9 +49,10 @@ where
 	C: ProvideRuntimeApi<Block>,
 	C: HeaderBackend<Block> + HeaderMetadata<Block, Error = BlockChainError> + 'static,
 	C: Send + Sync + 'static,
+	// TODO: Figure out why this doesn't compile
 	C::Api: substrate_frame_rpc_system::AccountNonceApi<Block, AccountId, Index>,
```

Comment:
I can't figure out how to fix the trait bounds here to get this to compile. I suspect it has to do with the types used (`Block`, `AccountId`, `Index`), but I'm not sure yet.

@ascjones you've touched an RPC thing or two in your day, if you have some time would you be able to see where I'm going wrong here?

Reply:
I wonder why the RPC-Extension (`rpc.rs`) is not part of the `substrate-parachain-template`. Maybe @NukeManDan knows?

Reply:
Figured it out, I was indeed using the wrong `Block` type and missing a trait bound in the `service`